### PR TITLE
Feat: Introduction of cucumber in mongo module

### PR DESCRIPTION
**Pull request by WLefever-Cegeka**
Introduced cucumber in the mongo module. 
Started with member entity.

### CODE REVIEW

File: ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: MemberRepository
+  As a user
+  I want get a stream of Members from the MemberSupplier
```
Comment (Tomvbe) [inline ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature:3]: Is it really a stream?

File: ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+  Scenario: Obtaining the members from first three fragments including the starting node
```
[the remark this answers is not in the document]
Comment (WLefever-Cegeka) [reply ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature]: updated

File: ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature (hunk 1 above)
Comment (WLefever-Cegeka) [reply ldes-server-infra-mongo/mongo-repository/src/test/resources/features/member.feature]: updated

**Review by Tomvbe — APPROVED** [review]

**Comment by Tomvbe** [comment]
Should we add tests for the other parts of the MemberRepository interface as well? memberExists, delete, addReference, etc. ?

**Comment by WLefever-Cegeka** [comment]
> Should we add tests for the other parts of the MemberRepository interface as well? memberExists, delete, addReference, etc. ?

Absolutely correct, will create an issue so that other methods in the interface are tested

**Pull request merged**